src/main/java/com/bugsnag/BugsnagLogger.java

```diff
@@ -0,0 +1,263 @@
+package com.bugsnag;
+
+import org.slf4j.Logger;
+import org.slf4j.Marker;
+import org.slf4j.helpers.MessageFormatter;
+
+public class BugsnagLogger implements Logger {
+    protected static final String LOGGER_NAME = "Bugsnag";
+
+    protected Client client;
+
+    public BugsnagLogger(Client client) {
+        this.client = client;
+    }
+
+    public String getName() {
+        return LOGGER_NAME;
+    }
+
+    public boolean isTraceEnabled() {
+        return false;
+    }
+
+    public void trace(String msg) {
+        // Trace not supported
```

Comment:
Once we have breadcrumbs, we can implement trace to leave a breadcrumb. Same for info.